css/backoffice/components/datatable/_datatableconfig.scss

```
@@ -22,6 +22,9 @@ $ibo-datatableconfig--settings-panel--option--margin-right: $ibo-spacing-200 !de
 }
 .ibo-datatableconfig--attributes-panel--per-page--input{
   margin: $ibo-datatableconfig--attributes-panel--per-page--input--margin-y $ibo-datatableconfig--attributes-panel--per-page--input--margin-x;
+  max-width: 4em;
+  @extend .ibo-input;
+  display: unset;
```

```suggestion
  display: initial;
```

We usually put `initial` instead to restore the default behavior.

---

Once this is fixed, you can:
- Rebase on support/3.2
- Change target branch to support/3.2
- Sqaush and merge (mind to keep the PR number)